README.md

```diff
@@ -1 +1,59 @@
-# chart-base
\ No newline at end of file
+# chart-base
+
+Base Helm chart for all applications.
+
+It adds below templates from [chart-library](https://github.com/hmcts/chart-library/) based on the chosen configuration:
+
+- [Deployment](https://github.com/hmcts/chart-library/tree/master#deployment)
+- [Horizontal Pod Auto Scaler](https://github.com/hmcts/chart-library/tree/master#hpa-horizontal-pod-auto-scaler)
+- [Ingress](https://github.com/hmcts/chart-library/tree/master#ingress)
+- [Pod Disruption Budget](https://github.com/hmcts/chart-library/tree/master#pod-disruption-budget)
+- [Service](https://github.com/hmcts/chart-library/tree/master#service)
+- [Deployment Tests](https://github.com/hmcts/chart-library/tree/master#smoke-and-functional-tests)
+
+## Language Settings
+
+As detailed in [chart-library](https://github.com/hmcts/chart-library/tree/master#language), applications can set `language` property to pick sensible defaults for a particular language.
+
+Example:
+```
+language: java
+```
+will override values from defaults with
+```
+java:
+  -----
```

Comment:
?

Reply:
was just denoting "whatever", looks like not the best way :)